.cirrus.yml

```diff
@@ -14,7 +14,7 @@ freebsd_task:
     - pkg install -y autoconf bison gmake re2c icu libiconv png freetype2 enchant2 bzip2 krb5 t1lib gmp tidyp libsodium libzip libxml2 libxslt openssl oniguruma pkgconf webp libavif
   script:
     - ./buildconf -f
-    - ./configure --prefix=/usr/local --enable-debug --enable-option-checking=fatal --enable-fpm --with-pdo-sqlite --without-pear --with-bz2 --with-avif --with-jpeg --with-webp --with-freetype --enable-gd --enable-exif --with-zip --with-zlib --enable-soap --enable-xmlreader --with-xsl --with-libxml --enable-shmop --enable-pcntl --enable-mbstring --with-curl --enable-sockets --with-openssl --with-iconv=/usr/local --enable-bcmath --enable-calendar --enable-ftp --with-kerberos --with-ffi --enable-zend-test --enable-dl-test=shared --enable-intl --with-mhash --with-sodium --enable-werror --with-config-file-path=/etc --with-config-file-scan-dir=/etc/php.d
```

Comment:
`--enable-werror` shoulud be re-added if possible.

Reply:
I m afraid it leads to some code changes ... I would suggest to either use another compiler than the system one (which is clang 17) e.g. clang 15 or dropping this setting.

Reply:
Some changes are ok, is it many? I suppose this is ok, given that we have warnings enabled for the other jobs.

Reply:
> Some changes are ok, is it many? I suppose this is ok, given that we have warnings enabled for the other jobs.

The bundled libbcmath is already a "mess" by itself, but even removing it there is other extensions involved like ffi ...

Reply:
Ok, then let's re-add the flag for master.